tools/testdata/check_format/header_order.cc.gold

```diff
@@ -19,7 +19,7 @@
 #include "common/api/os_sys_calls_impl.h"
 #include "common/common/utility.h"
 #include "common/common/version.h"
-#include "common/config/resources.h"
+#include "test/test_common/resources.h"
```

Comment:
This seems a regression, this isn't alpha order..

Reply:
reverted.